src/_posts/2020-02-20-remote-pairing-a-true-story.md

```diff
@@ -0,0 +1,68 @@
+ 
+## Pair Programming
+Pair programming is considered a Core Practice among those 12 practices enumerated within the XP scope, and is basically a social activity, is the point where two developers join to work together in the same tool they use to produce their craft, create enriched code that emerges from their interaction.
+
+Working in pairs relates deeply with other core practices such as [Code Ownership](https://ronjeffries.com/xprog/what-is-extreme-programming/#collective), [Code Standards](https://ronjeffries.com/xprog/what-is-extreme-programming/#coding) and even helps build the [Metaphor](https://ronjeffries.com/xprog/what-is-extreme-programming/#metaphor) of the system across the team.
+
+When you and the brand new team agree you'll embrace pair programming as a whole, all code to be sent into production is created by two people working together at a single computer.
+
+The best way to pair program is to just sit side by side in front of the monitor. [Slide the keyboard and mouse back and forth.](http://www.extremeprogramming.org/rules/pair.html)
+
+But, how does this last statement matches with the context presented in the paragraph above?
+
+## Pairing Remote is still pairing
+Pair Programming is a demanding activity and remote pairing is even more demanding, but it doesn't need to be even more exhausting, while it also can be done effectively. Here there is a compound of lessons learned that might help your team boost their pair programming experience:
+
+- Requires good and interrupted connections
+- Get comfortable headset for your team, avoid microphones that capture too much environmental noise.
+- There are dozens of tools available, but stick to a video conference that allows only one driver.
+- Ask questions and be prepared to explain, keep your pair engaged
+- Is not that simple to say "here" or "there", use the proper method or functions names, use the line numbers to reference a location in the file.
+- When information is lacking, use your video tool of reference to invite an external and evacuate doubts
+- Eventually you can try mob programming (for me didn't work, but I know many of my colleagues had great experiences doing mob remotely)
+
+The rest is basically use the common sense and the general pairing guidelines. I add some suggestions that might help you throughout your pairing experience:
+- Involve the whole team when doing planning meetings, make them feel part and organize the meetings taking remotes into account.
+- Have a refined backlog of work, which the whole involved team took part in its creation.
+- Share context, make them part of the discussions and own the topic (this is obviously beyond Pair Programming but is IMHO a key factor)
```

Comment:
I would replace 'them' with 'it' (if you are referring to the 'context').

I would avoid using IMHO and would write something like - 'but is, in my opinion, a key factor to build a shared understanding of the work in the team'.